code/modules/psionics/faculties/coercion.dm

```diff
@@ -248,18 +248,22 @@
 			to_chat(user, SPAN_CULT("Not even a psion of your level can speak to the dead."))
 			return
 
-		if (target.isSynthetic())
-			to_chat(user, SPAN_WARNING("This can only be used on living organisms."))
-			return
-
+		var/obj/item/organ/internal/augment/nlom/nlomaug
+		for (nlomaug in target)
```

Comment:
there are better ways of doing this, look into how welding fuel checks whether the target has the fuel aug

Reply:
not to bitch out or anything, but i would really prefer how it works right now